src/features/user/ManageProjects/conservationComponents/DetailedAnalysis.tsx

```diff
@@ -0,0 +1,532 @@
+        { id: 6, title: ready ? t('manageProjects:siteOwnerOther') : '', value: 'other', isSet: false }
+    ])
+    const landOwnershipType = [
+        {
+            label: ready ? t('manageProjects:siteOwnerPrivate') : '',
+            value: 'private',
+        },
+        {
+            label: ready ? t('manageProjects:siteOwnerPublic') : '',
+            value: 'public-property',
+        },
+        {
+            label: ready ? t('manageProjects:siteOwnerSmallHolding') : '',
+            value: 'smallholding',
+        },
+        {
+            label: ready ? t('manageProjects:siteOwnerCommunal') : '',
+            value: 'communal-land',
+        },
+        {
+            label: ready ? t('manageProjects:siteOwnerOwned') : '',
+            value: 'owned-by-owner',
+        },
+    ]
+    const ownershipType = [
+        {
+            label: ready ? t('manageProjects:tenure') : '',
+            value: 'tenure',
+        },
+        {
+            label: ready ? t('manageProjects:rent') : '',
+            value: 'rent'
+        },
+    ]
+    const [isUploadingData, setIsUploadingData] = React.useState(false)
+    const [errorMessage, setErrorMessage] = React.useState('')
+
+    const handleSetPlantingSeasons = (id: any) => {
+        const month = plantingSeasons[id];
+        const newMonth = month;
+        newMonth.isSet = !month.isSet;
+        const plantingSeasonsNew = plantingSeasons;
+        plantingSeasonsNew[id] = newMonth;
+        setPlantingSeasons([...plantingSeasonsNew]);
+    }
+
+    const handleSetSiteOwner = (id: any) => {
+        const owner = siteOwners[id - 1];
+        const newOwner = owner;
+        newOwner.isSet = !owner.isSet;
+        const newSiteOwners = siteOwners;
+        newSiteOwners[id - 1] = newOwner;
+        setSiteOwners([...newSiteOwners]);
+    }
+
+    React.useEffect(() => {
+        if (!projectGUID || projectGUID === '') {
+            handleReset(ready ? t('manageProjects:resetMessage') : '')
+        }
+    })
+
+
+    const { register, handleSubmit, errors, control, reset, setValue, watch } = useForm({ mode: 'onBlur' });
+
+    const onSubmit = (data: any) => {
+        setIsUploadingData(true)
+        const months = [];
+        for (let i = 0; i < plantingSeasons.length; i++) {
+            if (plantingSeasons[i].isSet) {
+                const j = i + 1;
+                months.push(j)
+            }
+        }
+
+        const owners = [];
+        for (let i = 0; i < siteOwners.length; i++) {
+            if (siteOwners[i].isSet) {
+                owners.push(siteOwners[i].value)
+            }
+        }
+        const submitData = {
+            projectMeta: {
+                location: data.location,
+                areaProtected: data.areaProtected,
+                employeeCount: data.employeeCount,
+                startingProtectionYear: data.startingProtectionYear.getFullYear(),
```

Comment:
- [ ] I got a crash here trying to add a new conservation project:

<img width="1583" alt="Bildschirmfoto 2022-01-15 um 10 35 10" src="https://user-images.githubusercontent.com/1532418/149618024-176964a7-8521-45ca-bee5-4d57afb1df1d.png">